conditional/blueprints/member_management.py

```diff
@@ -549,7 +550,7 @@ def introductory_project_submit():
 def get_member(uid):
     log = logger.new(user_name=request.headers.get("x-webauth-user"),
                      request_id=str(uuid.uuid4()))
-    log.info('api', action='submit introductory project results')
+    log.info('api', action="get %s's information" & uid)
```

Comment:
I believe you want `"get {}'s information".format(uid)` here. Or at least the `&` should be a `%`.